### PR TITLE
Ep cleanup saml proxy

### PR DESCRIPTION
**Pull request by edpaget**
Resolves department-of-veterans-affairs/vets-contrib#503

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,12 @@ This app provides a SAML SP and a SAML IdP that allows it to proxy SAML requests
 
 Requires Docker and/or Node.js > 8.
```
Comment (kalilsn) [inline README.md]: I actually had to downgrade to Node 8 to run this natively

Comment (edpaget) [reply README.md]: Oh yeah I think that's related to this: https://github.com/auth0/passport-wsfed-saml2/pull/111

Comment (edpaget) [reply README.md]: Updated the deps so it'll work with node 10

File: src/routes/constants.ts [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
 export const IDP_REDIRECT = '/samlproxy//idp/redirect';
 export const IDP_SIGN_IN = '/samlproxy/idp/signin';
-export const IDP_SIGN_OUT = '/samlproxy/idp/signout';
 export const IDP_SETTINGS = '/samlproxy/idp/settings';
-export const SP_SLO_URL = '/samlproxy/sp/saml/slo';
 export const SP_PROFILE_URL = '/samlproxy/sp/profile';
 export const SP_LOGIN_URL ='/samlproxy/sp/login';
-export const SP_LOGOUT_URL = '/samlproxy/sp/logout';
 export const SP_METADATA_URL = '/samlproxy/sp/metadata';
 export const SP_SETTINGS_URL = '/samlproxy/sp/settings';
```
Comment (patrickvinograd) [inline src/routes/constants.ts]: SP_SETTINGS_URL is unused.

File: src/routes/constants.ts [hunk 3]
```diff
@@ -1,14 +1,10 @@
 export const IDP_SSO = '/samlproxy/idp/saml/sso';
-export const IDP_SLO = '/samlproxy/idp/saml/slo';
 export const IDP_METADATA = '/samlproxy/idp/metadata';
 export const IDP_REDIRECT = '/samlproxy//idp/redirect';
 export const IDP_SIGN_IN = '/samlproxy/idp/signin';
-export const IDP_SIGN_OUT = '/samlproxy/idp/signout';
 export const IDP_SETTINGS = '/samlproxy/idp/settings';
```
Comment (patrickvinograd) [inline src/routes/constants.ts]: IDP_SETTINGS is unused

File: src/routes/constants.ts [hunk 4]
```diff
@@ -1,14 +1,10 @@
...
-export const SP_SLO_URL = '/samlproxy/sp/saml/slo';
 export const SP_PROFILE_URL = '/samlproxy/sp/profile';
```
Comment (patrickvinograd) [inline src/routes/constants.ts]: SP_PROFILE_URL is unused

File: src/routes/constants.ts [hunk 5]
```diff
@@ -1,14 +1,10 @@
 export const IDP_SSO = '/samlproxy/idp/saml/sso';
-export const IDP_SLO = '/samlproxy/idp/saml/slo';
 export const IDP_METADATA = '/samlproxy/idp/metadata';
 export const IDP_REDIRECT = '/samlproxy//idp/redirect';
 export const IDP_SIGN_IN = '/samlproxy/idp/signin';
```
Comment (patrickvinograd) [inline src/routes/constants.ts]: I think IDP_SIGN_IN is referenced but doesn't actually have a handler?

Comment (patrickvinograd) [reply src/routes/constants.ts]: nevermind, wasn't grepping with -i

**Pull request merged**